azure-kusto-ingest/package.json

```diff
@@ -22,9 +22,9 @@
     "prepublish": "npm run build",
     "example": "npm run build && node example.js",
     "lint": "npm run build && tslint --project tsconfig.json --quiet",
-    "test": "npm run build && mocha",
-    "e2e": "npm run build && mocha test/e2eTests/e2eTest.ts",
-    "allTests": "npm run build && mocha --timeout 240000 --recursive",
+    "test": "npm run build && mocha --parallel --recursive test/*.js",
```

Comment:
Please validate that this parallel will not make the tests flaky

Reply:
We will only run parallel tests when running locally, because mocha-junit-reporter doesn't work well with it.
The testPipeline doesn't run them in parallel